sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/fullfidelity/PartitionProcessorFactoryImpl.java

```diff
@@ -69,21 +69,12 @@ private static ChangeFeedStartFromInternal getStartFromSettings(
         FeedRangeInternal feedRange,
         ChangeFeedProcessorOptions processorOptions) {
 
-        if (!Strings.isNullOrWhiteSpace(processorOptions.getStartContinuation()))
-        {
+        //  TODO:(kuthapar) - how will customers use this? given now the continuation token in the lease will be the full json.
```

Comment:
In Spark I am still returning a string - basically the base64 encoded json or the ChangeFeedState - you should be able to use same approach